envoy.yaml

```diff
@@ -5,40 +5,59 @@ admin:
 
 static_resources:
   listeners:
-    - name: listener_0
-      address:
-        socket_address: { address: 0.0.0.0, port_value: 9092 }
-      filter_chains:
-        - filters:
-            - name: envoy.http_connection_manager
+  - name: listener_0
+    address:
+      socket_address: { address: 0.0.0.0, port_value: 9092 }
+    filter_chains:
+    - filters:
+      - name: envoy.http_connection_manager
+        config:
+          codec_type: auto
+          stat_prefix: ingress_http
+          stream_idle_timeout: 0s  #<-- ADDED FROM https://github.com/grpc/grpc-web/issues/361
+          route_config:
+            name: local_route
+            virtual_hosts:
+            - name: local_service
+              domains: ["*"]
+              routes:
+              - match: { prefix: "/census.Census/SpatialQuery" }
```

Comment:
This is incorrect. The prefix should match what's used in https://github.com/Project-Sustain/sustain-dataset-explorer/pull/3/files#diff-59b9f4f89c8fb22d12c0080f71585e9eR3

Reply:
Right mate, I'll change that and commit. Thanksfor catching it